typing/typemod.ml

```diff
@@ -943,7 +928,7 @@ let merge_constraint initial_env loc sg lid constr =
             by the caller. So what we do with the scope doesn't really matter. But
             making it local makes it unlikely that we will ever use the result of
             this function unfreshened without issue. *)
```

Comment:
I think you should delete this comment.  It explains the reason for `Make_local` on the next line, which no longer appears (and has also been deleted upstream, though it was in a slightly different place there because of our "lazy strengthening" rework).